curated_transformers/cli/quantize.py

```diff
@@ -0,0 +1,97 @@
+from io import BytesIO
+from pathlib import Path
+import spacy
+from spacy import Language
+from spacy.cli import app
+from thinc.api import Model, PyTorchShim
+import torch
+from torch.nn import Embedding, Module
```

Comment:
```suggestion
from torch.nn import Module
```
Unused.

Reply:
Fixed, but slightly differently. I also import `Linear` now and replaced the fully qualified `torch.nn.{Embedding,Linear}` later in the file.